ext/standard/url.c

```diff
@@ -321,12 +321,26 @@ PHPAPI php_url *php_url_parse_ex2(char const *str, size_t length, bool *has_port
 	return ret;
 }
 
+static void parse_url_decode_component(zval *zv, uri_component_read_mode_t read_mode)
```

Comment:
I figured that it's possible to add URL decoding support for parse_URL, so I added it. But let me know if it wasn't a good idea.